src/scripts/commands/metadata/sync.ts

```diff
@@ -47,27 +43,49 @@ export const syncMetadata = command({
             description: "Path to csv file with DHIS2 models to ignore (optional)",
             defaultValue: () => "",
         }),
+        action: option({
+            type: string,
+            long: "action",
+            description: "Action to perform (CREATE | CREATE_AND_UPDATE | DELETE)",
+            defaultValue: () => "",
```

Comment:
cmd-ts allows narrower types, example:

action: option({
            type: optional(oneOf(["CREATE", "CREATE_AND_UPDATE", "DELETE"] as const)),
            long: "action",
            description: "Action to perform (CREATE | CREATE_AND_UPDATE | DELETE)",
        }),

(and we don't use "" as fallback, which is typically a red flag)